crates/polars-expr/src/planner.rs

```diff
@@ -500,10 +500,7 @@ fn create_physical_expr_inner(
             let mut pd_group = ExprPushdownGroup::Pushable;
             pd_group.update_with_expr_rec(expr_arena.get(*evaluation), expr_arena, None);
 
-            let output_field_with_ctx = expr_arena
-                .get(expression)
-                .to_field_with_ctx(ctxt, &ToFieldContext::new(expr_arena, schema))?;
```

Comment:
Since it originally could have the `Context::Aggregation`, these data-types could differ, right? Is that no longer a problem?